http_requests.py

```diff
@@ -212,3 +220,30 @@ def delete(self, url: str, return_type=None, **kwargs):
                 return_type=None,
                 **kwargs
         )
+
+
+def _log_stacktrace_to_els(url: str):
+    if not ci.util._running_on_ci():
+        return  # early exit if not running in ci job
+
+    els_index = 'requests_stacktrace'
+    config_set = 'internal_active'
```

Comment:
do not hard-code cfg set. use cicd-cfg-set from ci.util; if no elasticsearch cfg is available, do not log